integration/setup/setup.go

```diff
@@ -35,30 +31,22 @@ const (
 func Setup(m *testing.M, config Config) {
 	ctx := context.Background()
 
-	// Perform teardown before execution.
-	{
-		err := teardown(ctx, config)
-		if err != nil {
-			os.Exit(1)
-		}
-	}
```

Comment:
We don't need teardown before execution because stack creation is idempotent now. This will save couple of minutes as a bonus.